dsgrid_project/datasets/historical/dsgrid_efs_2012_distributed_generation/dataset.json5

```diff
@@ -94,14 +88,12 @@
       description: "Weather year, 2012 only",
       file: "../../../dimensions/weather_years.csv",
       module: "dsgrid.dimension.standard",
-      name: "2012 Weather Year",
-      display_name: "2012 Weather",
+      name: "weather_year_2012",
       type: "weather_year",
     },
     {
       // Note: this does not match the project time because of it being period-ending, not period-beginning
-      name: "Time-2012-EST-hourly-periodEnding-noDST-noLeapDayAdjustment-total-UTChour1start",
-      display_name: "2012 Hourly EST",
+      name: "EST_hourly_2012",
```

Comment:
@elainethale You posted this string several times, and so I used it even though it is now lower case.